controllers/gce/controller/controller.go

```diff
@@ -330,7 +329,7 @@ func (lbc *LoadBalancerController) sync(key string) (err error) {
 	// Record any errors during sync and throw a single error at the end. This
 	// allows us to free up associated cloud resources ASAP.
 	var syncError error
-	if err := lbc.CloudClusterManager.Checkpoint(lbs, nodeNames, nodePorts); err != nil {
+	if err := lbc.CloudClusterManager.Checkpoint(lbs, nodeNames, allNodePorts); err != nil {
```

Comment:
I think line 305 will bite you if you no-op a lb create.

The original intent was even if you have an Ingress with all Services that don't have a nodePort, you should get an IP that sends you to the default backend. This was more for a live website that somehow transitioned from working lb with nodeport Services -> lb without nodeport Services, so users would at least see a 404 page. 

Since we're changing that assumption, we can also make that error a warning log line and return. Otherwise it'll retry on the error.